### DISCUSSION TITLE
Vector time 2nd derivative kernel

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#25300] [Q&A General] [answered]
I am getting the following error when attempting to use the vector time 2nd derivative kernel:  MooseVariableFE: Second time derivative of solution (`u_dotdot`) is not stored. Please set uDotDotRequested() to true in FEProblemBase before requesting `u_dotdot`.

What do i need to do in order to get the kernel to function?

**Comment by GiudGiud** [comment, marked as answer]
You have to access the FE problem from the kernel, in the constructor of the kernel, and do what the message says. 

If you grep for uDotDotRequested in the repository you can see how it s done

**Reply by zuckarm** [reply]
hello, can you please elaborate on this process? i want to set uDotDotRequested() as true, but i am unable to do so.

**Reply by GiudGiud** [reply]
Why are you unable to do so? Which object are you modifying?

**Reply by zuckarm** [reply]
i modified FEProblemBase.C file, there was a parameter _u_dotdot_requested(false), i changed it to true. is this the correct procedure?

**Reply by GiudGiud** [reply]
no you have to do this from an object. The object that needs this uses "coupledDotDot" to retrieve the 2nd time derivative and under the hood, _u_dotdot_requested should be set to true. 

It's fine for now though
